### PR TITLE
Add support for Docker's healthcheck feature

### PR DESCRIPTION
**Pull request by rmoff**
Healthcheck looks at Kafka Connect REST endpoint and declares Kafka Connect 'heathly' if it gets HTTP 200 from it. 

```
$ docker ps
CONTAINER ID        IMAGE                                    COMMAND                   CREATED             STATUS                   PORTS                              NAMES
8138c423eeb5        confluentinc/cp-kafka-connect:5.2.1      "bash -c 'echo \"Inst…"   2 minutes ago       Up 2 minutes (healthy)   0.0.0.0:8083->8083/tcp, 9092/tcp   kafka-connect
```

### CODE REVIEW

**Comment by gnschenker** [comment]
LGTM!

File: debian/kafka-connect-base/include/etc/confluent/docker/healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [[ $(curl -s -o /dev/null -w %{http_code} http://$CONNECT_REST_ADVERTISED_HOST_NAME:$CONNECT_REST_PORT/) = 200 ]]; then
+  echo "Woohoo! Kafka Connect is up!"
```
Comment (ybyzek) [inline debian/kafka-connect-base/include/etc/confluent/docker/healthcheck.sh]: Maybe change output to something more professional?

Comment (rmoff) [reply debian/kafka-connect-base/include/etc/confluent/docker/healthcheck.sh]: ¯\_(ツ)_/¯ it only would be seen if someone was using `docker inspect` to see the output of the healthcheck. Consider it an Easter Egg? 🥚

File: debian/kafka-connect-base/include/etc/confluent/docker/healthcheck.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  exit 0
+else 
+  echo -e $(date) "\tKafka Connect HTTP state: " $(curl -s -o /dev/null -w %{http_code} http://$CONNECT_REST_ADVERTISED_HOST_NAME:$CONNECT_REST_PORT/info) " (waiting for 200)"
```
Comment (ybyzek) [inline debian/kafka-connect-base/include/etc/confluent/docker/healthcheck.sh]: What if it's HTTPS?  Then this will never pass?

**Comment by rmoff** [comment]
Closing in favour of #769

**Pull request closed without merging**